### PR TITLE
Update Synology Drive Client to 3.0.1-12667

### PR DESCRIPTION
**Pull request by Keiishu**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19672)

### CODE REVIEW

File: manifests/s/Synology/DriveClient/3.0.1-12667/Synology.DriveClient.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Synology.DriveClient
+PackageVersion: 3.0.1-12667
+PackageLocale: en-US
+Publisher: Synology Inc.
```
Comment (OfficialEsco) [inline manifests/s/Synology/DriveClient/3.0.1-12667/Synology.DriveClient.locale.en-US.yaml:7]:
```suggestion
PackageVersion: 7.0.1.12667
PackageLocale: en-US
Publisher: Synology
```

File: manifests/s/Synology/DriveClient/3.0.1-12667/Synology.DriveClient.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Synology.DriveClient
+PackageVersion: 3.0.1-12667
```
Comment (OfficialEsco) [inline manifests/s/Synology/DriveClient/3.0.1-12667/Synology.DriveClient.installer.yaml:5]:
```suggestion
PackageVersion: 7.0.1.12667
```

File: manifests/s/Synology/DriveClient/3.0.1-12667/Synology.DriveClient.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Synology.DriveClient
+PackageVersion: 3.0.1-12667
```
Comment (OfficialEsco) [inline manifests/s/Synology/DriveClient/3.0.1-12667/Synology.DriveClient.yaml:5]:
```suggestion
PackageVersion: 7.0.1.12667
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Comment by Keiishu** [comment]
> 
> 
> image

Where does this come from? I only took the official Synology website as a reference, so I must say it is confusing.
The download link is for the version "3.0.1-12667", not "3.0.1-12667".
image

And the official company name is not referred as "Synology", but "Synology Inc.".
image

**Review by ItzLevvie — APPROVED** [review]

**Comment by Dennis14e** [comment]
Why was the version number changed from 3.0.1 to 7.0.1?
See https://www.synology.com/de-de/releaseNote/SynologyDriveClient

**Comment by ItzLevvie** [comment]
cc @Dennis14e: Windows Package Manager requires the version to be from the Control Panel.

Please update your application's version number if it's incorrect.

**Comment by OfficialEsco** [comment]
Ask the developers why they can't make a consistent version name.

**Comment by Keiishu** [comment]
> 
> 
> Why was the version number changed from 3.0.1 to 7.0.1?
> See [synology.com/de-de/releaseNote/SynologyDriveClient](https://www.synology.com/de-de/releaseNote/SynologyDriveClient)

> 
> 
> image

**Comment by Dennis14e** [comment]
Oh okay, that's weird from Synology.
Thanks for the clarification. @ItzLevvie @OfficialEsco @Keiishu

**Pull request merged**